### DISCUSSION TITLE
Total column verification using GridStat

### DISCUSSION THREAD

**Discussion by WillySedlacek-16WS** [dtcenter/METplus#2446] [Configuration] [answered]
In the GridStat configuration file, how should we set the forecast field level when attempting to verify forecasts over the total column rather than at a particular level? We've tried setting level = "FULL" and level = "TOTAL" without success. Our parameter of interest is total column cloud cover/fraction.

Thank you!

**Comment by CPKalb** [comment]
Hi:

The way to specify levels depends on the format of your data.  What format is your input data?  It would also be helpful to see the output from wgrib1/wgrib2 (if your data is grib1/2), or ncdump (if your data is netCDF).  Would you be able to paste that into the discussion?

Thanks,
Christina

**Reply by WillySedlacek-16WS** [reply]
Hi, Christina, 

The format of our input data is grib1. Would I be able to email you an example of the wgrib1 output? We're not permitted to share the data publicly. 

Thank you, 
Willy

**Reply by CPKalb** [reply, marked as answer]
Hi Willy:

You can upload an example to our FTP site, using the [instructions](https://github.com/dtcenter/METplus/discussions/954).  Without seeing the output from wgrib I can't give a definitive answer.  But, level = L0 would be one try.

Christina

**Reply by WillySedlacek-16WS** [reply]
Christina, 

We tried using level = L0 today, and that proved successful. It's still good to know about the FTP site in case we run into additional questions down the road. Thank you for your help!

**Reply by CPKalb** [reply]
Okay, great!  Thanks for marking the answer.  I'm going to lock the discussion to prevent future posts.  If you have additional questions, feel free to start up a new discussion.

Christina
